src/external/ldap.m4

```diff
@@ -80,7 +80,8 @@ AC_CHECK_MEMBERS([struct ldap_conncb.lc_arg],
                    [AC_DEFINE([HAVE_LDAP_CONNCB], [1],
                      [Define if LDAP connection callbacks are available])],
                    [AC_MSG_WARN([Found broken callback implementation])],
-                   [])],
+                   [AC_DEFINE([HAVE_LDAP_CONNCB], [1],
+                     [Define if LDAP connection callbacks are available])])],
```

Comment:
Hi,

thank you for the patch, I would like to suggest to add something like `... always set when cross-compiling` to the descriptions in lines 81 and 84.

bye,
Sumit

Reply:
Done.